src/components/Filters/README.md

```diff
@@ -0,0 +1,481 @@
+---
+name: Filters
+category: Lists and tables
+keywords:
+  - filters
+  - filtering
+  - filter control
+  - resource list
+  - index
+  - list filter
+  - table
+---
+
+# Filters
+
+Filters is a composite component that filters the items of a list or table.
+
+Merchants use filters to:
+
+- view different subsets of items in a list or table
+- filter by typing into a text field
+- filter by selecting filters or promoted filters
+
+The way that merchants interact with filters depends on the components that you decide to incorporate. In its simplest form, filters includes a text field and different filters. What the filters are and how they are exposed to merchants is flexible.
+
+---
+
+## Accessibility
+
+The filters component relies on the accessibility features of multiple other components:
+
+- [Text field](https://polaris.shopify.com/components/forms/text-field)
+- [Button](https://polaris.shopify.com/components/actions/button)
+- [Popover](https://polaris.shopify.com/components/overlays/popover)
+- [Sheet](https://polaris.shopify.com/components/overlays/sheet)
+- [Collapsible](https://polaris.shopify.com/components/behavior/collapsible)
+
+### Maintain accessibility with custom features
+
+Since custom HTML can be passed to the component for additional actions, ensure that the filtering system you build is accessible as a whole.
+
+All merchants must:
+
+- be able to identify and understand labels for all controls
+- be notified of state changes
+- be able to complete all actions with the keyboard, efficiently and easily
+
+---
+
+## Best practices
+
+The filters component should:
+
+- help reduce merchant effort by promoting the filters that are most commonly used
+- include no more than 2 or 3 promoted filters
+- consider small screen when deciding the number filters to include in a popover
+- let merchants select more than one filter from within the same category. For example, in the payment filter category, merchants can combine paid, partially paid, and unpaid
```

Comment:
We don't always do this. For example, with order list filters, you can add only one filter from the Status category at a time. 

```suggestion
- In most cases, let merchants select more than one filter from within the same category. For example, from the Payment status filter category, merchants can combine the Payment paid, Payment unpaid, and Payment partially paid filters. 
```

@sarahill Do you think we should mention the rationale for when to not allow merchants to combine filters from the same category, like we do with the Status filters? If yes, what is the rationale?

Reply:
Yeah I’m not sure we even want to include this. I think this presumes too much about a consumers data. Specifically, it would be for a filter category that has multiple options.